scene/debugger/scene_debugger.cpp

```diff
@@ -1710,7 +1714,11 @@ void RuntimeNodeSelect::_send_ids(const Vector<Node *> &p_picked_nodes, bool p_i
 		return;
 	}
 
+#ifdef _3D_DISABLED
+	const int limit = max_selection - selected_ci_nodes.size();
+#else
 	const int limit = max_selection - (selected_ci_nodes.size() + selected_3d_nodes.size());
+#endif // _3D_DISABLED
```

Comment:
```suggestion
	int limit = max_selection - selected_ci_nodes.size();
#ifndef _3D_DISABLED
	limit -= selected_3d_nodes.size();
#endif // _3D_DISABLED
```
Is fine for this to not be a constant.